Sources/CoreCommands/Options.swift

```diff
@@ -84,8 +84,8 @@ public struct LocationOptions: ParsableArguments {
     @Option(name: .customLong("destination"), help: .hidden, completion: .directory)
     public var customCompileDestination: AbsolutePath?
 
-    @Option(name: .customLong("experimental-cross-compilation-sdks-path"), help: .hidden, completion: .directory)
-    var ccSDKsDirectory: AbsolutePath?
+    @Option(name: .customLong("experimental-destinations-path"), help: .hidden, completion: .directory)
+    var destinationsDirectory: AbsolutePath?
```

Comment:
nit: I would keep the "cross compilation" in the variable name here to make it easy to reason about